paddle/fluid/operators/detail/sendrecvop_utils.cc

```diff
@@ -45,6 +46,13 @@ void SerializeToByteBuffer(const std::string& name, framework::Variable* var,
   void* payload = nullptr;
   size_t payload_size;
   ProtoEncodeHelper e(static_cast<char*>(buf), 1024);
+  // Note: normally the profiler is enabled in 1 trainer, hence only
```

Comment:
How to ensure that only one trainer is enabled?

Reply:
see vgg_fluid example. we can use args.task_index == 0 to verify this.